notebooks/Setting up a labeling task.ipynb

```diff
@@ -128,18 +128,14 @@
     "\n",
```

Comment:
<p> Is it possible to remove the entire dependency on firestore here and completely depend on the csv?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>yup</p>